### PR TITLE
Feat modules

### PR DESCRIPTION
**Pull request by mixcas**
- Add modules metaboxes
- Scaffold and style a little
- Add control for modules videos

FYI: We are not tracking the mouse position to scrub thru the video. I'm saying this cuz a functionality like that was mentioned while planning. If there's time left at the end we would try to do it then.

### CODE REVIEW

File: lib/site-options.php [hunk 1]
```diff
@@ -59,6 +59,49 @@ function igv_register_theme_options_metabox() {
     ),
   ) );
 
+  // Modules
+  $modules_group = $front_page_options->add_field( array(
```
Comment (varffs) [inline lib/site-options.php:63]: I think this should be a post type. It seems enough to be classes as a type of post esp in the future with another iteration of this website they might even get their own pages each.

File: src/js/main.js [hunk 2]
```diff
@@ -183,4 +186,30 @@ Site.Menu = {
...
+    // TODO: Launch enter animation
+
+    _this.bind();
+  },
+
+  bind: function() {
+    var _this = this;
+
+    // Bind mouse over
+    _this.$videos.on('mouseover.videoModule', function(event) {
```
Comment (varffs) [inline src/js/main.js:202]: they never play on mobile?

File: src/js/main.js [hunk 3]
```diff
@@ -165,6 +166,8 @@ Site.WhatIsVideo = {
 };
 
 Site.Menu = {
+  pointerPosition: undefined,
```
Comment (varffs) [inline src/js/main.js]: this doesnt do anything?

**Review by varffs — COMMENTED** [review]
A few questions. Mainly if post type or site option. Esp as I already did Demos as a post type they prob need to be consistently one or the other

File: src/js/main.js (hunk 3 above)
Comment (mixcas) [reply src/js/main.js]: will remove

File: lib/site-options.php (hunk 1 above)
Comment (mixcas) [reply lib/site-options.php:63]: we should leave for the next iteration

File: src/js/main.js (hunk 2 above)
Comment (mixcas) [reply src/js/main.js:202]: idk what te mobile strategy is

**Comment by varffs** [comment]
I don't really agree with having a mixed data approach in master. But for speed I'm going to merge this then we can fix later.

**Pull request merged**